src/components/ArticleVotes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { patchArticleById } from '../api';
+import '../styling/articleVotes.css';
+
+function ArticleVotes({ article_id, votes }) {
+  const [articleVotes, setArticleVotes] = useState(votes);
+  const [err, setErr] = useState(null);
+
+  const handleVote = (e) => {
+    let inc_votes = 0;
+    if (e.target.innerText === '👎') {
+      inc_votes = -1;
+    } else if (e.target.innerText === '👍') {
+      inc_votes = 1;
+    }
+    setArticleVotes((currVotes) => currVotes + inc_votes);
+    patchArticleById(article_id, inc_votes)
+      .then(() => {})
+      .catch((err) => {
+        setErr('Something went wrong, please try again!');
```

Comment:
Think there needs some retrospective modification of vote state if the request fails?